easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.8.1-foss-2020a.eb

```
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'CD-HIT'
+version = '4.8.1'
+
+homepage = 'http://weizhong-lab.ucsd.edu/cd-hit/'
```

The homepage seems to have moved
```suggestion
homepage = 'http://weizhongli-lab.org/cd-hit/'
```